lighthouse-cli/performance-experiment/server.js

```diff
@@ -94,8 +97,19 @@ function requestHandler(request, response) {
 
 function reportRequestHandler(request, response) {
   try {
+    const id = request.parsedUrl.query.id || defaultReportId;
```

Comment:
lets make a little `getSelectedId(request)` method for this and the other handlers